src/babel_utils.py

```diff
@@ -227,9 +227,12 @@ def write_compendium(synonym_list,ofname,node_type,labels={},extra_prefixes=[]):
                 nw['identifiers'] = [ {k[0]:v for k,v in nids.items()} for nids in node['identifiers']]
                 outf.write( nw )
                 synonyms = synonym_factory.get_synonyms(node)
-                if len(synonyms) > 0:
-                    for p,o in synonyms:
-                        sfile.write(f'{node["identifiers"][0]["identifier"]}\t{p}\t{o}\n')
+                synonyms.sort(key=lambda x:len(x))
```

Comment:
`synonyms` is a set(), so it doesn't have a `.sort()` method (giving the error: `'set' object has no attribute 'sort'`). I think you'll need to use something like https://stackoverflow.com/a/58820959/27310 to sort it.